### ISSUE TITLE
Make the tool more responsive and mobile friendly

### ISSUE THREAD

**Issue by NdibeRaymond** [tonythomas01/gdrive-to-commons#31]
I tried using this tool on a mobile device and I noticed that the fonts are way too small that I basically had to pinch and zoom to be able to read the words.

I think it is important to make it mobile friendly

**Comment by NdibeRaymond** [comment]
@tonythomas01 if this looks like a valid feature request to you, I would love to handle it

**Comment by tonythomas01** [comment]
Yes please. Lets start by creating a phabricator task and linking it here ? I think we should keep all our communication there.
